sparkbox/designsystemssurvey
# DES-112: React housekeeping

React formatting to clear out console warnings.

To test:
1. Review the console locally on `feat/2021`
2. Switch to this branch and check the console warnings
